### PR TITLE
primary PR - readme and configuration files are created

### PR DESCRIPTION
**Pull request by yilmazbahadir**
Since the node-monitoring-agent manual setup instructions PR is approved&merged, I am creating this PR which is more relevant and preceding this one https://github.com/nemgrouplimited/symbol-node-monitoring-agent/pull/1 . Keeping the old one as an option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+
+Please visit the [documentation page](https://docs.symbolplatform.com/guides/network/supernode-enrollment.html#manually) for the instructions.
+
+*Please note that you can use the following command to generate the agent certificates.
+
+```
+bash ./cmds/init.sh
+```
+
+This will create the certificates in the 'certs' directory.
```
Comment (segfaultxavi) [inline README.md:14]:
```suggestion
This will create the certificates in the `certs` directory.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
-# symbol-node-monitoring-agent
-Symbol node monitoring agent 
+# Symbol Node Monitoring Agent
+Symbol node monitoring agent is a small rest application that runs inside a catapult server box exercising the node testing. It is mandatory to have this application running either as part of symbol-bootstrap or manually colocated with the Symbol Node in order to be part of Symbol Node Rewards Programme.
+
+## Installation & Running
+
+Please visit the [documentation page](https://docs.symbolplatform.com/guides/network/supernode-enrollment.html#manually) for the instructions.
+
+*Please note that you can use the following command to generate the agent certificates.
```
Comment (segfaultxavi) [inline README.md:8]:
```suggestion
# Symbol Node Monitoring Agent
The Symbol node monitoring agent is a small REST application that runs inside a catapult server machine performing the node testing. It is mandatory to have this application running (deployed by either symbol-bootstrap or manually) in order to be part of the Symbol Node Rewards Programs.

## Installation & Running

Please visit the [documentation page](https://docs.symbolplatform.com/guides/network/supernode-enrollment.html#manually) for the instructions.

You can use the following command to generate the agent certificates:
```

File: agent.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
+NODE_PRIVATE_KEY={{{NODE_PRIVATE_KEY}}}
+; Replace host-name with the public host name of your node
+REST_GATEWAY_URL=http://node-hostname:3000
+REWARD_PROGRAM=SuperNode
+LOGGER_FILE=logs/agent.log
+CERTS_KEY_FILE=./certs/agent-key.pem
+CERTS_CERT_FILE=./certs/agent-crt.pem
+;you need to download the CA cert file as instructed in the Symbol Documentation Page and place in the `certs` directory
```
Comment (segfaultxavi) [inline agent.properties:8]:
```suggestion
; You need to download the CA cert file as instructed in the Symbol Documentation Page (see README.md) and place in the `certs` directory
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

**Comment by fboucquez** [comment]
Nice @yilmazbahadir , I would use this one now. Next week when we split the agent will update with how to compile and run from the source code.

**Pull request merged**